src/constants/Game.ts

```diff
@@ -0,0 +1,5 @@
+const NUMBER_LENGTH = 3;
+const START = 1;
+const END = 9;
```

Comment:
해당 코드는 객체로 관리해주심이 더 좋을거같아요!!
```suggestion
const INPUT_OPTION = {
  length: 3,
  start: 1,
  end: 9,
}
```